core/src/main/java/org/polypheny/db/tools/RoutedAlgBuilder.java

```diff
@@ -78,7 +75,7 @@ public RoutedAlgBuilder values( Iterable<? extends List<RexLiteral>> tupleList,
 
 
     @Override
-    public RoutedAlgBuilder scan( Iterable<String> tableNames ) {
+    public RoutedAlgBuilder scan( List<String> tableNames ) {
```

Comment:
Please adjust to entityNames. Or is this only for relational?

Reply:
Only relational, renamed method and parent accordingly.